contents/data_vis_makie_glmakie.md

```diff
@@ -1,13 +1,120 @@
 ## GLMakie.jl {#sec:glmakie}
 
-For the most common plots in 3d we will use the following `peaks` functions
+Visualising data in 3D is also a common practice to gain insight from your data. 
+A simple plot include of course, lines and points. So we will start with those and since we already know how layouts work, we will put that into practice. 
+
+### Scatters and lines
+
+For scatter plots we have two options, the first one is `scatter(x,y,z)` and the second one is `meshscatter(x,y,z)`. 
```

Comment:
A was between "scale in the axis direction" and 
"elongates in the axis directions", but I left the first one because it conveys how much elongations is applied to the markers,  "the scale".